src/devices/cpu/z80/z80.cpp

```diff
@@ -478,43 +362,44 @@ uint8_t z80_device::rm_reg(uint16_t addr)
 /***************************************************************
  * Read a word from given memory location
  ***************************************************************/
-inline void z80_device::rm16(uint16_t addr, PAIR &r)
+inline void z80_device::rm16(u16 addr, PAIR &r)
 {
-	r.b.l = rm(addr);
-	r.b.h = rm(addr+1);
+	r.b.l = data_read(addr);
+	T(m_memrq_cycles);
+	r.b.h = data_read(addr+1);
+	T(m_memrq_cycles);
 }
 
 /***************************************************************
  * Write a byte to given memory location
  ***************************************************************/
-void z80_device::wm(uint16_t addr, uint8_t value)
+inline void z80_device::data_write(u16 addr, u8 value) {
+	m_data.write_byte(translate_memory_address((u32)addr), value);
+}
+
+inline void z80_device::wm(u16 addr, u8 value)
 {
-	// As we don't count changes between read and write, simply adjust to the end of requested.
-	if(m_icount_executing != MTM) T(m_icount_executing - MTM);
-	m_data.write_byte(translate_memory_address(addr), value);
-	T(MTM);
+	data_write(addr, value);
+	T(m_memrq_cycles);
 }
```

Comment:
You’ve got an inconsistent mix of K&R and Allman style on two consecutive functions here.  Please make it use Allman style consistently.